### PR TITLE
convert packaging v1 to v2 [untested]

### PR DESCRIPTION
**Pull request by kay0u**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by kay0u** [comment]
!testme

**Comment by kay0u** [comment]
!testme

File: manifest.toml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+license = "MIT"
+website = "https://gitlab.com"
+demo = "https://gitlab.com/explore"
+admindoc = "https://docs.gitlab.com/"
+code = "https://gitlab.com/gitlab-org/omnibus-gitlab - https://gitlab.com/gitlab-org/gitlab"
+cpe = "???" # FIXME: optional but recommended if relevant, this is meant to contain the Common Platform Enumeration, which is sort of a standard id for applications defined by the NIST. In particular, Yunohost may use this is in the future to easily track CVE (=security reports) related to apps. The CPE may be obtained by searching here: https://nvd.nist.gov/products/cpe/search. For example, for Nextcloud, the CPE is 'cpe:2.3:a:nextcloud:nextcloud' (no need to include the version number)
+fund = "???" # FIXME: optional but recommended (or remove if irrelevant / not applicable). This is meant to be an URL where people can financially support this app, especially when its development is based on volunteers and/or financed by its community. YunoHost may later advertise it in the webadmin.
+
+[integration]
+yunohost = ">= 4.2.3"
```
Comment (ericgaspar) [inline manifest.toml:22]:
```suggestion
yunohost = ">= 11.1.15"
```

**Comment by kay0u** [comment]
!testme

**Comment by kay0u** [comment]
!testme

**Comment by kay0u** [comment]
!bookwormtestme

**Comment by kay0u** [comment]
!testme

**Comment by kay0u** [comment]
!testme

**Comment by kay0u** [comment]
!testme

File: manifest.toml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    # this is a generic question - ask strings are automatically handled by Yunohost's core
+    type = "path"
+    default = "/gitlab"
+
+    [install.admin]
+    # this is a generic question - ask strings are automatically handled by Yunohost's core
+    type = "user"
+
+    [install.init_main_permission]
+    help.en = "If your GitLab instance is public, anyone can see your public repositories."
```
Comment (ericgaspar) [inline manifest.toml:45]:
```suggestion
    help.en = "If your GitLab instance is set to visitors, anyone can see your public repositories."
```

File: manifest.toml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    help.fr = "Si votre instance GitLab est publique, tout le monde pourra voir vos dépôts publics."
```
Comment (ericgaspar) [inline manifest.toml:46]:
```suggestion
    help.fr = "Si votre instance GitLab est définie sur visiteurs, tout le monde pourra voir vos dépôts publics."
```

**Review by ericgaspar — COMMENTED** [review]
I believe closing ports is handled by the core in package v2...
https://github.com/YunoHost-Apps/gitlab_ynh/blob/35b1f86d803bf587c858c3da0c5dd27d5c0ae7a0/scripts/remove#L58-L67

**Comment by kay0u** [comment]
> I believe closing ports is handled by the core in package v2...
> https://github.com/YunoHost-Apps/gitlab_ynh/blob/35b1f86d803bf587c858c3da0c5dd27d5c0ae7a0/scripts/remove#L58-L67

Thanks for the review !

**Pull request merged**